pyproject.toml

```diff
@@ -0,0 +1,142 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=64.0.0",
+    "tomli  ; python_version < '3.11'",
+    "versioneer>=0.24",
+]
+
+[project]
+name = "dask-cuda"
+dynamic = [
+    "version",
+]
+description = "Utilities for Dask and CUDA interactions"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license= { text = "Apache-2.0" }
+requires-python = ">=3.8"
+dependencies = [
+    "dask >=2022.9.2",
+    "distributed >=2022.9.2",
+    "pynvml >=11.0.0",
+    "numpy >=1.18.0",
```

Comment:
Bumped to [align with Dask's `numpy` requirement]( https://github.com/dask/dask/blob/2022.11.0/setup.py#L15 )